src/config/custom.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
 
 module.exports = {
-  timeout: 80 * 1000
+  timeout: 40 * 1000
```

Comment:
I have seen many many tests that take longer than 5 seconds on both CI and locally so I am not sure this will just result in tons of test manually extending the timeout, which is worse imho than setting a relative high value here